core/variable_map.js

```diff
@@ -26,7 +26,8 @@
 
 goog.provide('Blockly.VariableMap');
 
-goog.require('Blockly.VariableModel');
```

Comment:
This file still uses Blockly.VariableModel...

Reply:
Fixed.